docs/features/techdocs/README.md

```diff
@@ -117,7 +58,47 @@ providers are used.
 
 [Reach out to us](#feedback) if you want to request more platforms.
 
-## Tech Stack
+## Project roadmap
+
+### **Ongoing work 🚧**
+
+**Beta release** -
+[Milestone](https://github.com/backstage/backstage/milestone/29)
+
+- It should be possible and easy to use TechDocs in most environments across
+  organizations.
+- Minimal bugs, better error handling and scalable backend and frontend.
+- Documentation Search
+- TechDocs Homepage with basic features
+
+### **Future work 🔮**
+
+**General Availability (GA) release** -
+[Milestone](https://github.com/backstage/backstage/milestone/30)
+
+- Bugs are rare, TechDocs APIs are stable and scales easily in large
+  organizations.
+- Better integration with
+  [Scaffolder V2](https://github.com/backstage/backstage/issues/2771) (e.g. easy
+  to choose and plug docs with Software Templates).
+- Possible to configure several aspects about TechDocs e.g. URL, homepage,
+  theme.
```

Comment:
* the first bullet uses an exempli gratia in parenthesis, but the second leaves it open as part of the sentence? suggest to move the second bullet part starting with `e.g.` into parens also?
* also what does it mean to "plug docs with" ? wonder if that's missing something

Reply:
👍 

Reworded to "plug documentation template with Software Templates". Idea is "Composing templates together (#docs-as-code) usecase" from https://github.com/backstage/backstage/issues/2771

On GitHub, when we create a new repository, we get to choose the type of .gitignore file and License type. Similarly, using Scaffolder one will create a new Library, a new Backend service, a new CLI and then choose from a list of available "Documentation templates" available in the organization. In Spotify, our technical writers have several such documentation templates available for different use-cases (which we plan to make public with the Software Templates Marketplace work https://github.com/backstage/backstage/issues/3383).